### ISSUE TITLE
Choose checklist using inner benchmark title

### ISSUE THREAD

**Issue by mpreisler** [OpenSCAP/scap-workbench#8]
In most cases this should be more convenient than choosing by checklist ID.

Original trac ticket: https://fedorahosted.org/scap-workbench/ticket/178

**Comment by mpreisler** [comment]
SSG provides source datastream now. Re-planning this sooner for 0.8.8 for nicer GUI support with default content.

**Comment by mpreisler** [comment]
This is a non-issue right now because we only show the combobox when there are multiple checklists.

Since this is a RFE I am postponing this until 1.0.0.

**Comment by mpreisler** [comment]
Non-issue still. No major content provider uses multiple checklists in a single datastream yet.

**Comment by mpreisler** [comment]
Still no use-case for this.

**Comment by matejak** [comment]
This is a valid request, related to https://bugzilla.redhat.com/show_bug.cgi?id=1457466
